corona-warn-app/cwa-app-android
# Global 7 day infections and hospitalisation incidents (EXPOSUREAPP-9833)

## Features
New combined card for 7 day infection and hospitalisation incidents.

<img width="355" alt="Screenshot 2021-10-11 at 15 45 02" src="https://user-images.githubusercontent.com/64849422/136792224-613d8ae7-134b-4914-b0f0-668cdab5a4aa.png">

## Jira
[Ticket](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-9833)
[Story](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-9449)

## How to test:
1) Run the application
2) Check the card
3) Enable accessibility features "Text to speech" and listen the card
